### PR TITLE
Populate versioned_input in find ER

### PR DESCRIPTION
**Pull request by anandjakhaniya**

### CODE REVIEW

File: backend/src/main/java/ai/verta/modeldb/experimentRun/subtypes/VersionInputHandler.java [hunk 1]
```diff
@@ -381,4 +386,63 @@ public VersionInputHandler(
         },
         executor);
   }
+
+  public InternalFuture<Map<String, VersioningEntry>> getVersionedInputs(Set<String> runIds) {
```
Comment (conradoverta) [inline backend/src/main/java/ai/verta/modeldb/experimentRun/subtypes/VersionInputHandler.java:410]: Same comment to add documentation here. The map and the last compose are very not clear to me

Comment (anandjakhaniya) [reply backend/src/main/java/ai/verta/modeldb/experimentRun/subtypes/VersionInputHandler.java:410]: Try to explain the code via adding comment in code

Comment (conradoverta) [reply backend/src/main/java/ai/verta/modeldb/experimentRun/subtypes/VersionInputHandler.java:410]: This method doesn't seem to be called by anything.

Comment (anandjakhaniya) [reply backend/src/main/java/ai/verta/modeldb/experimentRun/subtypes/VersionInputHandler.java:410]: it was there in commit https://github.com/VertaAI/modeldb/pull/2206/commits/bdbd5f02fd05ae328b25168cb298d29da17a54cd but due to merge conflicts it was removed. Now added removed code

**Review by conradoverta — APPROVED** [review]

**Pull request merged**